IoT/wled.1m.js

```diff
@@ -0,0 +1,102 @@
+#!/usr/local/bin/node
+//  -*- coding: utf-8 -*-xbar
+//  <xbar.title>WLED</xbar.title>
+//  <xbar.version>v1.0</xbar.version>
+//  <xbar.author>Billy Jacoby</xbar.author>
+//  <xbar.author.github>billyjacoby</xbar.author.github>
+//  <xbar.desc>Show and manage WLED devices from the menu bar.</xbar.desc>
+//  <xbar.dependencies>node,wled,axios</xbar.dependencies>
+
+//  Please install the npm dependencies in a folder named "wled" - I'm open to better ways of doing this too...
+
+// <xbar.var>string(VAR_HOST=""): WLED IP Address or hostname</xbar.var>
+const axios = require("./wled/node_modules/axios");
+const bitbar = require("./wled/node_modules/bitbar");
```

Comment:
maybe this should be `xbar` now?

Reply:
The NPM package seems to have not been renamed yet - i think it works the same way but there's no `xbar` package.

The documentation for NodeJS plugins still recommends using the ```bitbar``` plugin too.